examples/step-55/doc/results.dox

```diff
@@ -0,0 +1,166 @@
+<h1>Results</h1>
+
+<h3> Errors </h3>
+
+<table align="center" border="1">
+  <tr>
+    <th>&nbsp;</th>
+    <th>L2 Velocity</th>
+    <th>Rate</th>
+    <th>L2 Pressure</th>
+    <th>Rate</th>
+    <th>H1 Velocity</th>
+    <th>Rate</th>
+  </tr>
+  <tr>
+    <td>3D, 3 global refinements</td>
+    <td>0.000670888</td>
+    <td align="center">-</td>
+    <td>0.0036533</td>
+    <td align="center">-</td>
+    <td>0.0414704</td>
+    <td align="center">-</td>
+  </tr>
+  <tr>
+    <td>3D, 4 global refinements</td>
+    <td>8.38E-005</td>
+    <td>8.0073235678</td>
+    <td>0.00088494</td>
+    <td>4.1283024838</td>
+    <td>0.0103781</td>
+    <td>3.9959530164</td>
+  </tr>
+  <tr>
+    <td>3D, 5 global refinements</td>
+    <td>1.05E-005</td>
+    <td>8.0008689923</td>
```

Comment:
No need to provide rates to 11 digits. Using 8.0 or 8.01 for the rate is entirely sufficient. Same elsewhere in these tables.